server/container_update_resources.go

```diff
@@ -51,3 +56,57 @@ func toOCIResources(r *pb.LinuxContainerResources) *rspec.LinuxResources {
 		// TODO(runcom): OOMScoreAdj is missing
 	}
 }
+
+func (s *Server) updateContainerResources(ctx context.Context, c *oci.Container, resources *rspec.LinuxResources) (retErr error) {
+	if err := updateContainerSpec(c, resources); err != nil {
+		return err
+	}
+
+	defer func() {
+		if retErr != nil {
```

Comment:
I feel I'm missing something. Which piece of code is setting this value?

Reply:
because we return named error `(retErr error)` this variable set to latest returned err

Reply:
Nevermind, got it